sky/clouds/gcp.py

```diff
@@ -141,9 +137,51 @@ def region_zones_provision_loop(
             assert len(accelerators) == 1, accelerators
             acc = list(accelerators.keys())[0]
             acc_count = list(accelerators.values())[0]
-            regions = service_catalog.get_region_zones_for_accelerators(
+            acc_regions = service_catalog.get_region_zones_for_accelerators(
                 acc, acc_count, use_spot, clouds='gcp')
+            if instance_type == 'TPU-VM':
+                regions = acc_regions
+            else:
+                vm_regions = service_catalog.get_region_zones_for_instance_type(
```

Comment:
Good question. To my understanding, it will never happen in our current code base. While it is possible that `instance_type` and `accelerators` are None at the same time, `instance_type` cannot be None when `accelerators` is not None.

However, for just in case, I added
```python
if instance_type is None:
    regions = acc_regions
```

Reply:
That makes sense! Thanks!